.github/workflows/pythonapp.yml

```diff
@@ -15,6 +15,8 @@ jobs:
         python-version: 3.8
     - name: Install tox and coverage
       run: pip install tox coverage
+    - name: Install libsndfile1-dev
```

Comment:
is that related to moviepy?

Reply:
yeah, it's in the documentation of moviepy. It won't install otherwise.